src/server/pps/cmds/cmds.go

```diff
@@ -1598,6 +1598,36 @@ func Cmds(mainCtx context.Context, pachCtx *config.Context, pachctlCfg *pachctl.
 	validatePipeline.Flags().StringVarP(&pipelinePath, "file", "f", "", "A JSON file (url or filepath) containing one or more pipelines. \"-\" reads from stdin (the default behavior). Exactly one of --file and --jsonnet must be set.")
 	commands = append(commands, cmdutil.CreateAliases(validatePipeline, "validate pipeline", pipelines))
 
+	rerunPipeline := &cobra.Command{
+		Use:   "{{alias}} <pipeline>",
+		Short: "Rerun a pipeline.",
+		Long:  "This command is used to rerun an existing pipeline.",
+		Example: "\t- {{alias}} foo \n" +
+			"\t- {{alias}} foo --reprocess\n" +
+			"\t- {{alias}} foo --project bar\n",
+		Run: cmdutil.RunFixedArgs(1, func(args []string) error {
```

Comment:
I am not sure if `RunFixedArgs` is correct here.

Reply:
Makes sense here since we're always passing a pipeline to the command